fbw-a32nx/src/systems/fmgc/src/components/fms-messages/TurnAreaExceedance.ts

```diff
@@ -28,6 +29,10 @@ abstract class TurnAreaExceedance implements FMMessageSelector {
     }
 
     process(deltaTime: number): FMMessageUpdate {
+        if (!this.guidanceController.hasGeometryForFlightPlan(FlightPlanIndex.Active)) {
+            return FMMessageUpdate.NO_ACTION;
+        }
```

Comment:
This should go through the normal path so the message is recalled if it existed before.